client/components/account-balances/balance-tooltip.tsx

```diff
@@ -3,29 +3,165 @@
 			buttonLabel={ label }
+			maxWidth={ maxWidth }
 		/>
 	);
 };
 
-export default BalanceTooltip;
+export const TotalBalanceTooltip: React.FC< TotalBalanceTooltipProps > = ( {
+	balance,
+} ) => {
+	return (
+		<BalanceTooltip
+			label={ `${ fundLabelStrings.total } tooltip` }
+			maxWidth={ '315px' }
+			content={
+				<>
+					<>
+						{ interpolateComponents( {
+							mixedString: __(
+								'{{bold}}Total balance{{/bold}} combines both pending funds (transactions under processing) and available funds (ready for deposit). {{learnMoreLink}}Learn more{{/learnMoreLink}}',
+								'woocommerce-payments'
+							),
+							components: {
+								bold: <b />,
+								learnMoreLink: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a
+										rel="external noopener noreferrer"
+										target="_blank"
+										href={
+											documentationUrls.depositSchedule
+										}
+									/>
+								),
+							},
+						} ) }
+					</>
+					<InlineNotice
+						className="wcpay-account-balances__balances-total-balance-tooltip-notice"
+						isDismissible={ false }
+					>
+						{ __(
+							'Total balance = Available funds + Pending funds',
+							'woocommerce-payments'
+						) }
+					</InlineNotice>
+					<>
+						{ balance < 0 &&
+							interpolateComponents( {
+								mixedString: __(
+									'Negative account balance? {{discoverWhyLink}}Discover why.{{/discoverWhyLink}}',
+									'woocommerce-payments'
+								),
+								components: {
+									discoverWhyLink: (
+										// eslint-disable-next-line jsx-a11y/anchor-has-content
+										<a
+											rel="external noopener noreferrer"
+											target="_blank"
+											href={
+												documentationUrls.negativeBalance
+											}
+										/>
+									),
+								},
+							} ) }
+					</>
+				</>
+			}
+		/>
+	);
+};
+
+export const AvailableBalanceTooltip: React.FC< AvailableBalanceTooltipProps > = ( {
+	balance,
+} ) => {
+	return (
+		<BalanceTooltip
+			label={ `${ fundLabelStrings.available } tooltip` }
+			maxWidth={ balance < 0 ? '280px' : undefined }
+			content={
+				<>
+					<p>
+						{ interpolateComponents( {
+							mixedString: __(
+								'{{bold}}Available funds{{/bold}} have completed processing and are ready to be deposited into your bank account. {{learnMoreLink}}Learn more{{/learnMoreLink}}',
+								'woocommerce-payments'
+							),
+							components: {
+								bold: <b />,
+								learnMoreLink: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a
+										rel="external noopener noreferrer"
+										target="_blank"
+										href={
+											documentationUrls.depositSchedule
+										}
+									/>
+								),
+							},
+						} ) }
+					</p>
+					<p>
+						{ balance < 0 &&
+							interpolateComponents( {
+								mixedString: __(
+									'Negative account balance? {{discoverWhyLink}}Discover why.{{/discoverWhyLink}}',
```

Comment:
To me `account balance` refers to the `total` balance, and that linked doc page is primarily about why a merchant's total (aka "account") balance is negative (typically: more refunds or disputes than sales).

I don't think showing this info in the pending balance will cause issues, but I'm not convinced it's necessary. When the pending balance is negative, I _think_ (?) this is usually temporary due to how payouts work (possibly specific to instant?), and we don't explain much about this scenario on the help page.

Curious for thoughts from @csmcneill who has more experience of negative balances in practice!

Reply:
> When the pending balance is negative, I think (?) this is usually temporary due to how payouts work (possibly specific to instant?), and we don't explain much about this scenario on the help page.

That's my experience of it. I think that we can also include refunds and disputes in here, too, since those can also make an account go negative temporarily. If a balance is negative for an extended period of time, that's usually because we cannot debit the attached bank account or debit card.

I'm planning on including a new section in our document at https://woo.com/document/woopayments/deposits/deposit-schedule/#total-balance that focuses on the Total balance, what it means, and some nuances to keep an eye out for (e.g., it being lower than the available balance due to funds that are in flight).